### PR TITLE
fix: group trait telemetry

### PR DESCRIPTION
**Pull request by jmorganca**

### CODE REVIEW

File: internal/server/telemetry.go [hunk 1]
```diff
@@ -100,17 +100,22 @@ func (t *Telemetry) EnqueueHeartbeat() {
...
+			oid = org.ID.String()
 		}
 	}
 
-	t.Event(event, uid, properties...)
+	t.Event(event, uid, oid, properties...)
 }
 
-func (t *Telemetry) Event(event string, userId string, properties ...map[string]interface{}) {
+func (t *Telemetry) Event(event string, userId string, orgId string, properties ...map[string]interface{}) {
```
[the remark this answers is not in the document]
Comment (jmorganca) [reply internal/server/telemetry.go:118]: It's not an optional property and should be included in every event.

File: internal/server/telemetry.go [hunk 2]
```diff
@@ -147,13 +154,19 @@ func (t *Telemetry) User(id string, name string) {
...
 		return
 	}
 	err := t.Enqueue(analytics.Group{
-		GroupId:   id,
-		Traits:    analytics.NewTraits().SetName(name),
+		GroupId: id,
+		UserId:  userID,
+		Traits: map[string]interface{}{
+			"name":   name,
+			"$name":  name,
```
Comment (BruceMacD) [inline internal/server/telemetry.go:166]: What is the difference between `name` and `$name` here?

Comment (jmorganca) [reply internal/server/telemetry.go:166]: One is a "super property". Some tools require both

**Review by BruceMacD — APPROVED** [review]
looks good to me

**Review by mxyng — APPROVED** [review]

**Pull request merged**